dev/sparktestsupport/modules.py

```diff
@@ -113,6 +113,14 @@ def __hash__(self):
     ],
 )
 
+utils = Module(
```

Comment:
Moving it is because of 

<img width="545" alt="image" src="https://github.com/apache/spark/assets/1475305/cc40aa65-2c04-4ca6-9a34-3b3da30954c1">

Reply:
yeah, this is python :)